soc/arm/arm/musca_a/Kconfig.soc

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2018 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+choice
+prompt "ARM Musca SoCs"
+depends on SOC_SERIES_MUSCA
+
+config SOC_V2M_MUSCA_A
+	bool "ARM Cortex-M33 SMM-SSE-200 on V2M-MUSCA"
+	select CPU_CORTEX_M33
+	select CPU_HAS_ARM_SAU
```

Comment:
@galak @agross-linaro do you select, somewhere CPU_HAS_ARM_MPU? Have I missed it?

Reply:
I swear I added that.  But perhaps I lost it when I shifted things around.  Let me add it back.

Reply:
Sure :)

Reply:
done